apps/cyberstorm-storybook/stories/components/Pagination.stories.tsx

```diff
@@ -1,11 +1,11 @@
-import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { StoryFn, Meta } from "@storybook/react";
 import { Pagination } from "@thunderstore/cyberstorm";
 import React, { useState } from "react";
 
-const meta = {
+export default {
```

Comment:
Is there any specific reason to change the default export to happen here? For me it would be clearer if all the exports happened in one place, at the end of the file.

Reply:
Not that well founded reasoning. Did use component that was up to date on how imports & exports should be dealt with and also storybook forum post used example done like this so thought it would be best practices.
After further reading about how CSFv3 would work and readability I would also second to revert this change.